samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,149 @@
+# {{ cookiecutter.project_name }}
```

Comment:
Could you please update this doc based on the new README we're working on?

https://github.com/awslabs/aws-sam-cli/blob/0622b9bb9b601a6c8c1e4054c2a0c93c2e70c96b/samcli/local/init/templates/cookiecutter-aws-sam-hello-dotnet/%7B%7Bcookiecutter.project_name%7D%7D/README.md

It basically adds a new Appendix, SAM Logs section, brings an `event.json` file _and_ subsequently shares how to execute the function without API Gateway too, etc..